nym-vpnd/Cargo.toml

```diff
@@ -28,6 +28,9 @@ tracing.workspace = true
 tonic-health.workspace = true
 tonic-reflection.workspace = true
 
+# match the version required by tonic
+http = "0.2.12"
```

Comment:
Suggestion: if we need to match tonic, have a look to see if maybe they re-export these types there

Reply:
I had checked, but tonic seems not to reexport these types (unless I missed it)